pontoon/base/formats/lang.py

```diff
@@ -0,0 +1,133 @@
+import codecs
+import re
+from collections import namedtuple
+
+from parsimonious.grammar import Grammar
+from parsimonious.nodes import NodeVisitor
+
+from pontoon.base.utils import search
+
+
+LangComment = namedtuple('LangComment', ['content'])
+
+
+TAG_REGEX = re.compile(r'\{(ok|l10n-extra)\}')
+
+
+class LangEntity(object):
+    def __init__(self, string, translation, comments, tags):
+        self.string = string
+        self.translations = {None: translation}
+        self.comments = comments
+        self.fuzzy = False
```

Comment:
Correct, but if we don't do this then the sync code that handles setting the fuzzy flag or reading it in the database would throw AttributeErrors when we try to access the property. So we fake it out on the entity and never use it when reading or writing to the lang files.